tests_requirements.txt

```diff
@@ -1,3 +1,4 @@
+black
```

Comment:
Tests per se don't require black. The CI job requires it yes, this is not a test, but format checking.

Reply:
If you want people to be able to check that tests run locally, they'll need black.

Reply:
Or even to run `make format` before creating PRs

Reply:
Right, why is that in test dependency?

Reply:
how about renaming the file to not be specific to tests?

Reply:
the format-check is a test...

Reply:
it's required to run the tests (including format check) locally